extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -509,6 +541,21 @@ export class Bind {
     }
   }
 
+  /**
+   * Returns a copy of the global state for a given field-based expression,
+   * e.g. "foo.bar.baz".
+   * @param {string} expr
+   * @return {(?JsonObject|string|undefined)}
+   */
+  getStateCopy(expr) {
```

Comment:
Nit: Let's just call this `getState()`. There shouldn't be a use case for returning a non-copy.